<launch>
    <!-- <include file="$(find rm_cv)/launch/detector.launch"/> -->
    <node pkg="self_aiming" name="server_node" type="server_node.py" output="screen"/>
    <node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
    <node pkg="self_aiming" name="smith_predictor_with_pid" type="smith_predictor_with_pid.py" output="screen"/>
</launch>
